go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/NYTimes/gziphandler v1.1.1
 	github.com/alicebob/miniredis/v2 v2.30.4
 	github.com/aquasecurity/bolt-fixtures v0.0.0-20200903104109-d34e7f983986
-	github.com/aquasecurity/defsec v0.92.0
+	github.com/aquasecurity/defsec v0.92.1-0.20230925225724-4b598f8ffb36
```

Comment:
note: update this to latest defsec when https://github.com/aquasecurity/defsec/pull/1460 is merged